tools/cellxgene_census_builder/entrypoint.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
 python3 -m cellxgene_census_builder .
+BUILDER_STATUS=$?
+# On error, log dmesg to aid troubleshooting
+# Note: requires docker --privileged option
+if [[ $BUILDER_STATUS -ne 0 ]]; then dmesg; fi
```

Comment:
nit 1: is there any way to test if privileged and only call if affirmative?

nit 2: I"m not sure we need all of the kernel log.  `dmesg | tail -100` is probably sufficient for our OOM/etc purposes

Reply:
> nit 1: is there any way to test if privileged and only call if affirmative?

From within the container, it's (reportedly) not possible. Other than just trying the command and handling the error. Since this is only run in an builder erroring state, then container will exit with a non-zero status either way. So I'm just going to leave it as is.

> 
> nit 2: I"m not sure we need all of the kernel log. `dmesg | tail -100` is probably sufficient for our OOM/etc purposes

Done! Though I took poetic license on the tail size.